docs/guide/field_type_reference.md

```diff
@@ -710,7 +726,7 @@ Using the variation Service, variations of the original image can be obtained. T
 
 #### Field Definition options
 
-The Image Field Type supports one FieldDefinition option: the maximum size for the file.
+The Image Field Type supports one `FieldDefinition` option: the maximum size for the file.
```

Comment:
This seems to be good place to add note saying that maximum size is rounded to 1MB (legacy storage limitation).